src/db.c

```diff
@@ -3509,12 +3509,24 @@ void init_char(struct char_data *ch)
     GET_EXP(ch) = 7000000;
 
     /* The implementor never goes through do_start(). */
-    GET_MAX_HIT(ch) = 500;
-    GET_MAX_MANA(ch) = 100;
-    GET_MAX_MOVE(ch) = 82;
+    GET_ADD(ch) = 100;
+    GET_STR(ch) = GET_LEVEL(ch);
+    GET_INT(ch) = GET_LEVEL(ch);
+    GET_WIS(ch) = GET_LEVEL(ch);
+    GET_DEX(ch) = GET_LEVEL(ch);
+    GET_CON(ch) = GET_LEVEL(ch);
+    GET_CHA(ch) = GET_LEVEL(ch);
+    GET_MAX_HIT(ch) = GET_LEVEL(ch) *10;
+    GET_MAX_MANA(ch) = GET_LEVEL(ch) *10;
+    GET_MAX_MOVE(ch) = GET_LEVEL(ch) *10;
     GET_HIT(ch) = GET_MAX_HIT(ch);
     GET_MANA(ch) = GET_MAX_MANA(ch);
     GET_MOVE(ch) = GET_MAX_MOVE(ch);
+    GET_HITROLL(ch) = GET_LEVEL(ch);
+    GET_DAMROLL(ch) = GET_LEVEL(ch);
+    GET_AC(ch) = -100;
+    GET_GOLD(ch) = 1000000; // convenience to test shops
+    SET_BIT_AR(PRF_FLAGS(ch), PRF_HOLYLIGHT);
```

Comment:
Are you sure these are changes you want to submit to the base game?  These look like customizations that you should keep to your copy of the code.

Reply:
I think the damroll and AC should be included in the base game for in-game testing purposes as Implementor and as a convienence to avoid using the "set" command. The other two, I suppose are not important. They were included as convienences. Of course if consensus or the powers that be disagree, then so be it. :)

Reply:
I just noticed:
 GET_AC(ch) = -100;
should be:
 GET_AC(ch) = GET_LEVEL(ch);

Reply:
negative AC is better than positive.

Reply:
True, however, GET_AC(ch) is set to a positive signed integer everywhere in the sourcecode. In file fight.c the compute_armor_class function is "supposed to" convert value of GET_AC(ch) to a negative signed integer, then add your dex bonus modifier "dex_app[GET_DEX(ch)].defensive", which is already a negative integer. 

Interestingly enough, I checked the output of that compute_armor_class function and it was not functioning correctly. One would expect the numeric output to equal the (negative value of GET_AC(ch)) + dex_app[GET_DEX(ch)].defensive, but that's not what it produces. The code fails to convert GET_AC(ch) to a negative integer. So rather than adding two negatives, we get a positive added to a negative.